README.rst

```diff
@@ -1,4 +1,5 @@
-.. image:: https://cloud.githubusercontent.com/assets/7521600/15992701/ef245fd4-30ef-11e6-992d-275c5ca7c3a0.jpg
+.. image:: https://cloud.githubusercontent.com/assets/5716520/24838296/a9cf5f04-1d45-11e7-855c-47b816ce1e09.png
+    :target: https://coala.io/
```

Comment:
Please make it to  `https://coala.io` instead `https://coala.io/`

Reply:
It also works fine  without any change but see the diffrence between these two http://api.coala.io/en/latest/Developers/Newcomers_Guide.html/  and http://api.coala.io/en/latest/Developers/Newcomers_Guide.html

Hope you will get my point

Reply:
That is irrelevant